app/javascript/components/applications/ApplicationPage.js

```diff
@@ -7,33 +7,38 @@ import mapProps from '~/utils/mapProps'
 import { mapApplication } from '~/components/mappers/soqlToDomain'
 import labelMapperFields from './application_details/applicationDetailsFieldsDesc'
 
-const buildActionLinkIfNecessary = (app) => {
-  if (!app.isLotterComplete && app.submissionType === 'Paper') {
-    return { title: 'Edit Application', link: appPaths.toApplicationEdit(app.id) }
-  } else {
-    return undefined
-  }
+const buildActionLinkIfNecessary = (app, fromPage) => {
+  const actions = []
+
+  if (!app.is_lottery_complete && app.application_submission_type === 'Paper')
+    actions.push(<a key='edit-application' href={appPaths.toApplicationEdit(app.id)} className='primary button tiny '>Edit Application</a>)
+
+  if (fromPage === 'new')
+    actions.push(<a key='add-new-application' href={appPaths.toApplicationNew(app.listing.id)} className='button tiny margin-left--half'>Add new application</a>)
+
+  return actions
 }
 
 const ApplicationPage = (props) => {
-  const { application } = props
+  const { application, fromPage} = props
```

Comment:
missing a space before bracket here. I think the linter isn't checking this. it's on my list to update the linter